meetup-wordpress-medellin.php

```diff
@@ -6,6 +6,24 @@
  * Version: 1.0.0
  */
 
+/**
+ * Función principal utilizada para hacer las verficaciones iniciales y
+ * configurar las acciones y filtros que soportan la funcionalidad del
+ * plugin.
+ *
+ * @since 1.0.0
+ */
+function mwpm_wordpress_configured() {
+    // El formulario para reportar entradas solo será mostrado en páginas y
+    // publicaciones individuales.
+    if ( is_single() || is_page() ) {
+        add_filter( 'the_content', 'mwpm_add_report_button_to_content' );
+        add_action( 'wp_enqueue_scripts', 'mwpm_enqueue_style' );
+        add_action( 'wp_footer', 'mwpm_enqueue_script' );
+    }
+}
+add_action( 'wp', 'mwpm_wordpress_configured' );
+
 function mwpm_generate_button_for_post( $post_id ) {
     $user_id = get_current_user_id();
```

Comment:
Estaba pensando en usar la información del usuario en la ventana modal, por eso dejé esta línea. creo que deberíamos usarla, en caso contrario removerla.

Reply:
+1, sería chévere usar el nombre y correo si lo tenemos disponible. Send the PR 🤟🏼